modules/reference/pages/systems/system-details/sd-software.adoc

```diff
@@ -1,4 +1,4 @@
 [[sd-software]]
-= SD Software
+= Software
 
-This tab and its subtabs allow you to manage the software on the system: patches (errata), packages and package profiles, software channel memberships, and migrations.
\ No newline at end of file
+The menu:Systems[Software] section allows you to manage software on the selected client.
```

Comment:
```suggestion
The menu:System Details[Software] section allows you to manage software on the selected client.
```
Maybe, we'd better say: "The Software section of the System Details page ..." (I'm not sure about the adoc decoration, though).